node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/DynamicDockerProvisioningTest.java

```diff
@@ -48,18 +48,17 @@
 
     /**
      * Test relocation of nodes that violate headroom.
-     *
+     * <p>
      * Setup 4 docker hosts and allocate one container on each (from two different applications)
      * No spares - only headroom (4xd-2)
-     *
+     * <p>
      * One application is now violating headroom and need relocation
-     *
-     *  Initial allocation of app 1 and 2 --> final allocation (headroom marked as H):
-     *
+     * <p>
+     * Initial allocation of app 1 and 2 --> final allocation (headroom marked as H):
+     * <p>
      * | H  |  H  | H   | H   |        |    |    |    |    |
```

Comment:
If you want this to render well in html you also need <pre> tags.
But nobody's going to look at the test javadoc so imho it was good without any tags ...

Reply:
Artifact of "alt-command l" - I'll be more carefull to not format javadoc in the future ;)